js/overrides.js

```diff
@@ -35,4 +35,6 @@
     'should send ignore region by selector with vg': {config: {branchName: 'universal-sdk'}}, // was skipped
     'should send floating region by coordinates with vg': {config: {branchName: 'universal-sdk'}}, // was skipped
     'should send accessibility regions by selector with vg': {config: {branchName: 'universal-sdk'}}, // was v1
+
+    'AUTproxy should proxy resources test manually only': {skip: true},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8974587)
```suggestion
    "AUTproxy should proxy resources test manually only": {skip: true},
```